dialoguekit/nlu/nlu.py

```diff
@@ -2,44 +2,49 @@
 
 from typing import List
 
-from dialoguekit.core.intent import Intent
-from dialoguekit.core.slot_value_annotation import SlotValueAnnotation
+from dialoguekit.core.annotation import Annotation
+from dialoguekit.core.dialogue_act import DialogueAct
 from dialoguekit.core.utterance import Utterance
-from dialoguekit.nlu.intent_classifier import IntentClassifier
-from dialoguekit.nlu.slot_annotator import SlotValueAnnotator
+from dialoguekit.nlu.annotator import Annotator
+from dialoguekit.nlu.dialogue_acts_extractor import DialogueActsExtractor
 
 
 class NLU:
     def __init__(
         self,
-        intent_classifier: IntentClassifier,
-        slot_annotators: List[SlotValueAnnotator],
+        dialogue_act_extractor: DialogueActsExtractor,
+        annotators: List[Annotator] = None,
```

Comment:
Would be nice to explain the difference between Annotator and DialogueActsExtractor in the file-level docstring (giving some examples of what these other-that-dialogueActs annotators do).